assets/javascripts/discourse/components/group-reports-nav-item.js.es6

```diff
@@ -0,0 +1,23 @@
+import { ajax } from "discourse/lib/ajax";
+
+export default Ember.Component.extend({
+  group: null,
+  showReportsTab: false,
+
+  checkForReports() {
+    const p1 = ajax(`/g/${this.group.name}/reports`);
```

Comment:
Trivial but you could avoid the `p1` variable here by returning `ajax(...).then()`

Reply:
👍